src/components/Match/MatchStory.jsx

```diff
@@ -146,14 +173,17 @@ class IntroEvent extends StoryEvent {
     this.game_mode = match.game_mode;
     this.region = match.region;
     this.date = new Date(match.start_time * 1000);
+    this.match_duration_seconds = match.duration;
   }
   format() {
     return formatTemplate(strings.story_intro, {
+      game_mode_article: ['A', 'E', 'I', 'O', 'U'].includes(strings[`game_mode_${this.game_mode}`].charAt(0)) ? 'an' : 'a',
```

Comment:
More generally I think the rule is to use "an" if the first syllable is a vowel sound.  For example, "hour" uses "an" even though the first letter is not a vowel.  I don't know if that applies to any of the game modes right now, but worth considering.

Reply:
'an' and 'a' should probably go into the string file

Reply:
Good call. Doesn't affect any of the current game modes, but I abstracted out an `articleFor(word)` method we can improve down the line if we ever get a game mode that breaks things (or want to use this elsewhere).